README.md

```diff
@@ -76,29 +76,29 @@ Conclusion:               # definitie van 1 eindconclusie
+1. [een doorklikbaar diagram](https://minbzk.github.io/ai-act-decisiontree/mermaid_graphs/decision-tree-main.html): in dit diagram is de beslishulp op een versimpelde manier weergeven. Dit diagram bevat de volgende componenten:
 
-    - donkerblauwe rechthoeken: dit zijn doorklikbare componenten die doorverwijzen naar een volgende categorie uit de beslisboom.
-    - lichtblauwe rechthoeken: dit zijn de 'labels' uit de beslisboom. De labels geven de karakteristieken die u door de beslisboom te doorlopen kunt verzamelen weer.
-    - donkerblauwe cirkels: dit zijn de vragen uit de beslisboom. Wanneer u op een cirkel klikt, verschijnt de volledige vraag in een pop-up op het scherm. Deze kan vervolgens weer weg worden geklikt door rechtsboven op het kruisje te klikken.
-    - witte hexagons met groene rand: dit zijn de conclusies uit de beslisboom. Wanneer u op een hexagon klikt, verschijnen de bij de conclusie behorende verplichtingen in een pop-up op het scherm.
+    - donkerblauwe rechthoeken: dit zijn doorklikbare componenten die doorverwijzen naar een volgende categorie uit de beslishulp.
+    - lichtblauwe rechthoeken: dit zijn de 'labels' uit de beslishulp. De labels geven de karakteristieken die u door de beslishulp te doorlopen kunt verzamelen weer.
+    - donkerblauwe cirkels: dit zijn de vragen uit de beslishulp. Wanneer u op een cirkel klikt, verschijnt de volledige vraag in een pop-up op het scherm. Deze kan vervolgens weer weg worden geklikt door rechtsboven op het kruisje te klikken.
+    - witte hexagons met groene rand: dit zijn de conclusies uit de beslishulp. Wanneer u op een hexagon klikt, verschijnen de bij de conclusie behorende verplichtingen in een pop-up op het scherm.
 
     U kunt terug naar het vorige diagram door de vorige pagina-knop in uw browser te gebruiken.
 
-2. een [overzichtsdiagram](https://minbzk.github.io/ai-act-decisiontree/mermaid_graphs/decision-tree-complete.html): in dit diagram is de gehele beslisboom gevat. Dit diagram is niet doorklikbaar. De vragen zijn gegroepeerd categorieen, weergeven in witte rechthoeken met groene randen.
+2. een [overzichtsdiagram](https://minbzk.github.io/ai-act-decisiontree/mermaid_graphs/decision-tree-complete.html): in dit diagram is de gehele beslishulp gevat. Dit diagram is niet doorklikbaar. De vragen zijn gegroepeerd categorieen, weergeven in witte rechthoeken met groene randen.
 
 Let op: de diagrammen kunnen het beste in Chrome, Safari of Microsoft Edge worden geopend.
 
 
 ## Frontend
 
-Om door de beslisboom te lopen is een visualizatie tool gemaakt. Met deze tool kunt u door de vragen lopen. De frontend is beschikbaar op deze [website](https://ai-act-decisiontree.apps.digilab.network).
+Om door de beslishulp te lopen is een visualizatie tool gemaakt. Met deze tool kunt u door de vragen lopen. De frontend is beschikbaar op deze [website](https://ai-act-decisiontree.apps.digilab.network).
```

Comment:
Would we want to keep the same url or should we also change this? For now only these texual changes seems fine but then we would also need to change the ai-validation-infrastructure repo

Reply:
I made a new ticket for the full-change.